python/private/py_wheel.bzl

```diff
@@ -176,6 +176,10 @@ _other_attrs = {
         doc = "A string specifying the license of the package.",
         default = "",
     ),
+    "project_urls": attr.string_dict(
+        doc = ("A string dict specifying additional browsable URLs for the project and corresponding labels. " +
```

Comment:
Nit pick. Mention the key first and the label second. You have them reversed.

Reply:
Updated with better explanation about the way to specify.